documentation/meta/project_boards/issues.md

```diff
@@ -0,0 +1,44 @@
+# Issues project: Openverse Backlog
+
+The [Openverse Backlog](https://github.com/orgs/WordPress/projects/75) project
+board tracks all issues through their lifecycle, as they move from creation to
+completion. This board does not track any pull requests, so all workflows for
+this board are tied to events occurring for issues.
+
+## Event automations
+
+### Issue is created
+
+If a new issue is created, it is automatically added to the project board
+provided it does not contain the any label with the text "project".
```

Comment:
This workflow also sets the `Priority` custom field to the issue in the project so that we can then create a kanban board by priority. We can not create kanban board based on labels.

Reply:
Noted in b8f90c555e8b76e7ef7c9cd6ec0a01c0d2cf7417.